test/tests/e2e.js

```diff
@@ -84,36 +87,96 @@ describe( 'Publish a New Post', function() {
 		);
 	} );
 
 	step( 'Can return to reader', async function() {
-		await driver.get( loggedInUrl );
+		return await driver.get( loggedInUrl );
 	} );
 } );
 
-// TODO: Fixme: This test is failing with the latest Calypso, but manually testing the sequence actually works.
-/*describe( 'Can Log Out', function() {
+describe( 'Can Log Out', function() {
 	this.timeout( 30000 );
 
 	step( 'Can view profile to log out', async function() {
 		let navbarComponent = await NavBarComponent.Expect( driver );
-		await navbarComponent.clickProfileLink();
+		return await navbarComponent.clickProfileLink();
 	} );
 
 	step( 'Can logout from profile page', async function() {
 		const profilePage = await ProfilePage.Expect( driver );
-		await profilePage.clickSignOut();
+		return await profilePage.clickSignOut();
 	} );
 
 	step( 'Can see app login page after logging out', async function() {
 		return await LoginPage.Expect( driver );
 	} );
-} );*/
+} );
+
+describe( 'Can Sign up', function() {
+	this.timeout( 30000 );
+	const blogName = dataHelper.getNewBlogName();
+	const expectedBlogAddresses = dataHelper.getExpectedFreeAddresses( blogName );
+	const emailAddress = blogName + process.env.E2E_MAILOSAUR_INBOX;
+
+	step( 'Clear local storage', async function() {
+		await driver.executeScript( 'window.localStorage.clear();' );
+		return await driver.sleep( 3000 );
+	} );
+
+	step( 'Can navigate to Create account', async function() {
+		let loginPage = await LoginPage.Expect( driver );
+		await loginPage.hideGdprBanner();
+		await loginPage.openCreateAccountPage();
+		return await SignupStepsPage.Expect( driver );
+	} );
+
+	step( 'Can see the "Site Topic" page, and enter the site topic', async function() {
+		const signupStepsPage = await SignupStepsPage.Expect( driver );
+		return await signupStepsPage.aboutSite();
+	} );
+
+	step( 'Choose a theme page', async function() {
+		const signupStepsPage = await SignupStepsPage.Expect( driver );
+		return await signupStepsPage.selectTheme();
+	} );
+
+	step(
+		'Can search for a blog name, can see and select a free .wordpress address',
+		async function() {
+			const signupStepsPage = await SignupStepsPage.Expect( driver );
+			return await signupStepsPage.selectDomain( blogName );
+		}
+	);
+
+	step( 'Can see the plans page and pick the free plan', async function() {
+		const signupStepsPage = await SignupStepsPage.Expect( driver );
+		return await signupStepsPage.selectFreePlan();
+	} );
+
+	step( 'Can see the account page, enter account details and submit', async function() {
+		const signupStepsPage = await SignupStepsPage.Expect( driver );
+		return await signupStepsPage.enterAccountDetailsAndSubmit(
+			emailAddress,
+			blogName,
+			process.env.E2EPASSWORD
+		);
+	} );
+
+	step( 'Can then see the onboarding checklist', async function() {
+		const checklistPage = await ChecklistPage.Expect( driver );
+		return await checklistPage.isChecklistPresent();
+	} );
+
+	// 	TODO:delete account
```

Comment:
@Stojdza Did you intend to leave TO-DO/comment block?  Perhaps we can make this a GH issue instead, once this is merged?

Reply:
I was planning to leave TODOs, but GH issue sounds like a way better option! :) Thanks!